icons/arduino-white.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0"?> 
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" viewBox="0 0 24 24" width="24px" height="24px"><g transform="translate(31.5,31.5) scale(0.7,0.7)"><g fill="none" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal"><path d="M0,210v-210h210v210z" fill="none"/><g fill="#ecf0f1"><g id="surface1"><path d="M48.46154,56.53846c-26.84946,0 -48.46154,21.61208 -48.46154,48.46154c0,26.84946 21.61208,48.46154 48.46154,48.46154c19.52975,0 35.33654,-11.61058 46.1899,-22.46394c4.25932,-4.25932 7.60366,-8.45553 10.34856,-12.11538c2.74489,3.65986 6.08924,7.85607 10.34856,12.11538c10.85337,10.85337 26.66015,22.46394 46.1899,22.46394c26.84946,0 48.46154,-21.61208 48.46154,-48.46154c0,-26.84946 -21.61208,-48.46154 -48.46154,-48.46154c-19.52975,0 -35.33654,11.61058 -46.1899,22.46394c-4.19621,4.19621 -7.60366,8.23467 -10.34856,11.86298c-2.71334,-3.59675 -5.96304,-7.69832 -10.09615,-11.86298c-10.79026,-10.88491 -26.59706,-22.46394 -46.44231,-22.46394zM48.46154,72.69231c13.28275,0 25.61899,8.61328 34.83173,17.92067c7.22506,7.31971 9.84375,11.61058 11.61058,14.38702c-1.82992,2.83955 -4.57482,7.09886 -11.86298,14.38702c-9.33894,9.33894 -21.80138,17.92067 -34.57933,17.92067c-18.39393,0 -32.30769,-13.91376 -32.30769,-32.30769c0,-18.39393 13.91376,-32.30769 32.30769,-32.30769zM161.53846,72.69231c18.39393,0 32.30769,13.91376 32.30769,32.30769c0,18.39393 -13.91376,32.30769 -32.30769,32.30769c-12.77794,0 -25.24038,-8.58173 -34.57933,-17.92067c-7.28816,-7.28816 -10.03305,-11.54747 -11.86298,-14.38702c1.82993,-2.83954 4.57482,-7.09886 11.86298,-14.38702c9.33894,-9.33894 21.80138,-17.92067 34.57933,-17.92067zM155.98558,88.84615v10.60096h-10.60096v11.10577h10.60096v10.60096h11.10577v-10.60096h10.60096v-11.10577h-10.60096v-10.60096zM32.30769,99.44712v10.34856h32.30769v-10.34856z"/></g></g><path d="" fill="none"/><path d="" fill="none"/></g></g></svg>
```

Comment:
Looking at the svg code, it seems you've not optimized it, could you fix that? 🤔